dyn_em/module_initialize_fire.F

```diff
@@ -250,8 +250,8 @@ SUBROUTINE init_domain_rk ( grid &
    IF (sfc_init) THEN
     DO j = jts, jte
       DO i = its, ite
-          grid%xlat(i,j) = config_flags%fire_lat_init     !Ak/sk (35)
-         grid%xlong(i,j) = config_flags%fire_lon_init     !Ak/ak (-111)
+          grid%xlat(i,j) = 0. 
+         grid%xlong(i,j) = 0.
```

Comment:
@masih-e 
I looked in the `registry.fire` file. These fields are already assigned to zero:
```
grep -i fire_lat_init *
registry.fire:rconfig   real    fire_lat_init          namelist,fire         max_domains           0.      -      "fire_lat_init"    "latitude to start fire"           "degrees"
etotheipi:/Users/gill/Desktop/WRF_noahmp/Registry>grep -i fire_lon_init *
registry.fire:rconfig   real    fire_lon_init          namelist,fire         max_domains           0.      -      "fire_lon_init"    "longitude to start fire"          "degrees"
```

These run-time options are defined in the namelists in the em_real directory:
```
find . -name namelist\* -exec grep -iw fire_lat_init {} \; -print
!fire_lat_init      =   40.,    ! real   , initial fire latitude (deg)
 fire_lat_init      = 6.75e-3,  ! real   , initial fire latitude (deg)
./em_real/namelist.input.fire

find . -name namelist\* -exec grep -iw fire_lon_init {} \; -print
!fire_lon_init      = -105.,    ! real   , initial fire longitude (deg)
 fire_lon_init      = 6.70e-3,  ! real   , initial fire longitude (deg)
./em_real/namelist.input.fire
```

Perhaps there is another problem.

Reply:
I see. So those lines are fine. It's only the call set_ideal_coord() should be removed which is overwriting the 253 and 254.